src/commands/config/generate.ts

```diff
@@ -138,8 +155,7 @@ export default class ConfigGenerateCmd extends Command {
 		await this.validateOptions(options);
 
 		let resourceDeviceType: string;
-		let application: Awaited<ReturnType<typeof this.getApplication>> | null =
```

Comment:
How come and you decided to remove the explicit type?
Or was it balena-lint that removed it?
(I do realize that as soon as the assignment happens this has the correct types, just saying)

Reply:
This was a mistake, thanks!